src/arkode/arkode.c

```diff
@@ -3397,6 +3397,9 @@ int arkCheckTemporalError(ARKodeMem ark_mem, int* nflagPtr, int* nefPtr,
   ---------------------------------------------------------------*/
 sunbooleantype arkAllocVec(ARKodeMem ark_mem, N_Vector tmpl, N_Vector* v)
 {
+  /* return failure if N_VClone is not implemented */
+  if (!tmpl->ops->nvclone) { return SUNFALSE; }
```

Comment:
I should have thought of this when noting the check here but we should similarly check if `ops->nvdestroy` is defined in `arkFreeVec` in case `ARKodeFree` is called before `arkInit`